velox/exec/tests/PartitionedOutputTest.cpp

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/exec/PartitionedOutput.h"
+#include <gtest/gtest.h>
+#include "velox/exec/Task.h"
+#include "velox/exec/tests/utils/AssertQueryBuilder.h"
+#include "velox/exec/tests/utils/OperatorTestBase.h"
+#include "velox/exec/tests/utils/PlanBuilder.h"
+
+using namespace facebook::velox;
+using exec::test::AssertQueryBuilder;
+
+namespace facebook::velox::exec::test {
+
+class PartitionedOutputTest : public OperatorTestBase {
+ protected:
+  static std::string generateString(size_t size, char value) {
+    std::string result;
```

Comment:
I think you can just use one of std::string's constructors:

https://en.cppreference.com/w/cpp/string/basic_string/basic_string

`std::string(size, value)`

Reply:
Nice